modules/conversion-center/base/widget-link-in-bio-base.php

```diff
@@ -1381,21 +1375,12 @@ protected function add_style_background_controls(): void {
 			'background' => [
 				'default' => 'classic',
 			],
-			'size' => [
-				'default' => 'cover',
-			],
-			'position' => [
-				'default' => 'center center',
```

Comment:
This looks good. 

One question: How is the control displayed by default? As 'Position: Default' or as 'Position: Center Center'.

I believe that the expected behavior is 'Position: Default', and that the default styling should be handled with SCSS.

See the default settings of the Container:
![Screenshot 2024-05-20 at 14 27 22](https://github.com/elementor/elementor/assets/94352322/3ba7faff-013f-4484-89c7-1ccc25048a58)

I assume that you are on the right track, because I see similar coding here: https://github.com/elementor/elementor/blob/5777fc0d3955163df07a59f37ca9f844f8866a42/includes/controls/groups/background.php#L142-L141

Reply:
@willhowat Thank you for the above feedback.
I would love to hear your feedback on this one.

Reply:
@hein-obox this is displayed as in your example 'Default'. We have not set any defaults in the CSS at this time but it's a sensible ask. I'll raise a separate task to get this added